src/ast/nodes/MemberExpression.ts

```diff
@@ -262,7 +262,8 @@ export default class MemberExpression extends NodeBase implements DeoptimizableE
 			this.object instanceof Identifier &&
 			this.scope.findVariable(this.object.name).isNamespace
```

Comment:
For style points, you could split this condition and assign the return value of `this.scope.findVariable` to a local variable to avoid the double lookup.

Reply:
Sure, I've added the refactoring.